examples/plt_cfg.yaml

```diff
@@ -43,8 +43,9 @@ plots:
 
   scan:
     - signal: bjets
-      efficiency: 0.70
```

Comment:
So the code does not break if additional arguments are provided here? That is something that would be good to have

Reply:
Good shout, added a `check_config` function and relevant unit test.